documents/Libraries/stdlib/sx-glsl/mx_gamma18_to_linear_color3.glsl

```diff
@@ -0,0 +1,5 @@
+void mx_gamma18_to_linear_color3(vec3 _in, out vec3 result)
+{
+    vec3 gamma = vec3(1.799999952316284, 1.799999952316284, 1.799999952316284);
```

Comment:
These I think you can make constants here. Seems to be a few more in other places as well.